docs/Release_Guide/metviewer_official.rst

```diff
@@ -8,4 +8,14 @@ Create a new vX.Y.Z official release from the develop branch.
 .. include:: release_steps/open_release_issue.rst
 .. include:: release_steps/clone_project_repository.rst
 .. include:: release_steps/checkout_develop_branch.rst
-.. include:: release_steps/create_release_feature_branch.rst
+.. include:: release_steps/metviewer/update_version_official.rst
+.. include:: release_steps/update_release_notes_official.rst
+.. include:: release_steps/metviewer/update_docs_tagged.rst
+.. include:: release_steps/merge_release_issue.rst
```

Comment:
The following should be added 
.. include:: release_steps/rotate_authorship.rst
above this line ".. include:: release_steps/merge_release_issue.rst"
but you will need to pull in the changes from develop in order to see/use this file as it was newly added.

Reply:
I am not aware and don't perform this step for METviewer.